tasks/taxon_id/task_kraken2.wdl

```diff
@@ -0,0 +1,116 @@
+version 1.0
+
+task kraken2_pe {
+  input {
+    File read1
+    File read2
+    File kraken2_db
+    String samplename
+    String docker = "quay.io/staphb/kraken2:2.1.2-no-db"
+
+    String? kraken2_args = ""
+    String? classified_out = "classified#.fastq"
```

Comment:
@kapsakcj does the pe classified out have a `#`? Not seeing that same notation in the se task.

Reply:
If paired-end kraken2 splits on the `#` for `_1` and `_2`. And if single-end, it does not split on `#`, so the `#` would end of un the name for single-end.

More details here: https://github.com/DerrickWood/kraken2/blob/master/src/classify.cc#L680-L696

Reply:
Thanks, @rpetit3!